### PR TITLE
#2576 Group all ITs by type so as to test them group by group

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2576)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-connector/src/test/scala/com/island/ohara/connector/PerfGroup.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.island.ohara.connector;
+
+public interface PerfGroup {}
```
Comment (saivirtue) [inline ohara-connector/src/test/scala/com/island/ohara/connector/PerfGroup.java]: It seems this class is not used?

Comment (chia7712) [reply ohara-connector/src/test/scala/com/island/ohara/connector/PerfGroup.java]: You are right

**Comment by chia7712** [comment]
retry it

**Comment by chia7712** [comment]
retry it

**Comment by chia7712** [comment]
retry it

**Comment by chia7712** [comment]
retry it

**Comment by chia7712** [comment]
retry it

**Comment by chia7712** [comment]
retry

**Comment by chia7712** [comment]
retry

**Comment by chia7712** [comment]
retry

**Comment by chia7712** [comment]
retry

**Comment by chia7712** [comment]
retry

**Comment by chia7712** [comment]
retry it

**Comment by chia7712** [comment]
retry it

**Comment by chia7712** [comment]
retry it

**Comment by chia7712** [comment]
retry it

**Comment by chia7712** [comment]
retry otherIT

**Comment by chia7712** [comment]
retry sshCollie

**Comment by chia7712** [comment]
retry sshConnector

**Pull request merged**